src/dex/hashflow/constants.ts

```diff
@@ -20,3 +20,37 @@ export const HASHFLOW_GAS_COST = 100_000;
 
 export const HASHFLOW_MIN_SLIPPAGE_FACTOR_THRESHOLD_FOR_RESTRICTION =
   new BigNumber('0.001');
+
+export const RESTRICT_79_TTL_MS = 60 * 10 * 1000; // 10 mins
```

Comment:
@JackieJoo I would you use a bit more reasonable naming here - for example RESTRICT_BELOW_MINIMUM_AMOUNT_TTL_MS